azure-auth-helper/src/main/java/com/microsoft/azure/auth/AzureAuthHelper.java

```diff
@@ -149,9 +158,108 @@ public static AzureCredential readAzureCredentials(File file) throws IOException
         throw new UnsupportedOperationException("Not implemented");
     }
 
+    /**
+     * Get azure token credential from $HOME/.azure/azure-secret.json(created by `mvn azure:login`)
+     *
+     * @return the azure token credential can be used in Azure SDK.
+     * @throws IOException when there are some IO errors.
+     */
+    public static AzureTokenCredentials getMavenAzureLoginCredentials() throws IOException {
+        final AzureCredential credentials = readAzureCredentials(getAzureSecretFile());
+        final AzureEnvironment env = getAzureEnvironment(credentials.getEnvironment());
+        return getMavenAzureLoginCredentials(credentials, env);
+    }
+
+    /**
+     * Get azure credential from AzureCredential instance.
+     *
+     * @param credentials the azure credential
+     * @param env the azure environment
+     * @return the azure token credential can be used in Azure SDK.
+     * @throws IOException when there are some IO errors.
+     */
+    public static AzureTokenCredentials getMavenAzureLoginCredentials(AzureCredential credentials, AzureEnvironment env) throws IOException {
```

Comment:
@NotNull for credentials?

Reply:
@NotNull will have no actual check if the parameter is null?